addon.xml

```diff
@@ -4,8 +4,10 @@
        version="0.1.7a3"
        provider-name="Plex">
     <requires>
-        <import addon="xbmc.python" version="2.25.0"/>
-        <import addon="script.module.requests" version="2.3.0"/>
+        <import addon="xbmc.python" version="3.0.0"/>
```

Comment:
This should remain at the old version. I can push to the Kodi 19 repo with just this version change.